### DISCUSSION TITLE
How to request HTTP asset in Maui Blazor WebView?

### DISCUSSION THREAD

**Discussion by zxyao145** [dotnet/maui#6917] [Q&A]
I will load a http image in blazor component, but I got error:

```Mixed Content: The page at 'https://0.0.0.0/' was loaded over HTTPS, but requested an insecure image 'http://xxxxx'. This request has been blocked; the content must be served over HTTPS.```

So, is there any way to solve this problem (except upgrading to https) ?

**Comment by HugoVG** [comment]
Bump, same issue, upgrading to https brings more issues, so i would prefer being able to make HTTP calls to web api during development

**Reply by kirk-wgt** [reply]
Bump2.  Trying to pull images from an s3 bucket and thought that I was fighting with CORS issues but it seems that it's instead because my HTTP calls are getting automagically converted to HTTPS.

**Comment by a21354123** [comment]
+1
